cadasta/test/account_tests/test_registration.py

```diff
@@ -26,80 +24,61 @@ def test_new_registration(self):
             self.wd.find_css('#id_email').clear()
             self.wd.find_css('#id_email').send_keys(
                 test_username + "@example.com")
-            self.wd.find_css("#id_password1").clear()
-            self.wd.find_css("#id_password1").send_keys(test_password)
-            self.wd.find_css("#id_password2").clear()
-            self.wd.find_css("#id_password2").send_keys(test_password)
+            self.wd.find_css("#id_password").clear()
+            self.wd.find_css("#id_password").send_keys(test_password)
             self.wd.find_css("#id_full_name").send_keys('')
             action = ActionChains(self.wd)
             action.send_keys(Keys.TAB).send_keys(Keys.RETURN).perform()
 
             time.sleep(1)
             elems = self.wd.find_elements_by_xpath(
                 "//*[contains(text(), "
-                "'A user with that username already exists.')]")
+                "'A user with that username already exists')]")
             elems.extend(self.wd.find_elements_by_xpath(
                 "//*[contains(text(), "
                 "'Another user with this email already exists')]"))
             if len(elems) == 0:
                 username_available = True
-                Credentials().set_test_username(test_username)
-                Credentials().set_test_password(test_password)
-                Credentials().set_test_email(test_username + "@example.com")
+                # Credentials().set_test_username(test_username)
```

Comment:
Some tests require this behavior but this messes up subsequent tests. This is because with pytest, some class state is maintained between tests. I just commented these lines out while I haven't decided yet what to do replace these with, if ever I decide to replace them while debugging the other tests.